spring-cloud-dataflow-build/spring-cloud-dataflow-build-dependencies/pom.xml

```diff
@@ -52,6 +52,41 @@
 	</properties>
 	<dependencyManagement>
 		<dependencies>
+			<dependency>
+				<groupId>org.apache.tomcat.embed</groupId>
+				<artifactId>tomcat-embed-el</artifactId>
```

Comment:
Please read through this [previous comment](https://github.com/spring-cloud/spring-cloud-dataflow/pull/5795/files#r1594794522).